main.py

```diff
@@ -25,7 +26,10 @@ def main(input_code, input_lang, output_lang):
     elif input_lang == "java":
         gast = java_main.java_to_gast(input_code)
     elif input_lang == "bash":
-        gast = bash_main.bash_to_gast(input_code)
+        process = subprocess.Popen(['node', 'bash/code_to_gast/bash_main.js', input_code], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
this is fine for now, but feel like this should probably eventually live somewhere else